src/lib/Components/Inbox/Conversations/ZeroStateInbox.tsx

```diff
@@ -48,25 +55,59 @@ const Icon = styled(Image)`
   width: 40;
 `
 
-export default () =>
-  <CenteredView>
-    <VerticalLayout>
-      <Title>BUYING ART ON ARTSY IS SIMPLE</Title>
-      <HorizontalLayout>
-        <Icon source={require("../../../../../images/find.png")} />
-        <SmallHeadline>Follow artists and categories to find works you love.</SmallHeadline>
-      </HorizontalLayout>
-      <HorizontalLayout>
-        <Icon source={require("../../../../../images/contact.png")} />
-        <SmallHeadline>Contact galleries or bid in auctions to purchase the wor.</SmallHeadline>
-      </HorizontalLayout>
-      <HorizontalLayout>
-        <Icon source={require("../../../../../images/message.png")} />
-        <SmallHeadline>Find your ongoing conversations and bidding activity here.</SmallHeadline>
-      </HorizontalLayout>
-      <HorizontalLayout>
-        <Icon source={require("../../../../../images/pay.png")} />
-        <SmallHeadline>Easily process payment through our secure platform.</SmallHeadline>
-      </HorizontalLayout>
-    </VerticalLayout>
-  </CenteredView>
+export default class ZeroStateInbox extends React.Component<null, null> {
+  render() {
+    /** TODO: Maxim see if there's any other way  */
+    if (isPad) {
+      return (
+        <CenteredView>
+          <VerticalLayout>
+            <Title>BUYING ART ON ARTSY IS SIMPLE</Title>
+            <List>
```

Comment:
This is the only difference? you could do something like: `const Wrapper = isPad ? List : View` then use that inside this render function

Reply:
the thing is that `View` causes iPhone to be rendered differently. It should be `List` or _nothing_

Reply:
Ok just had another idea to try which worked. BRB